common/common.cpp

```diff
@@ -69,5 +69,28 @@ string JoinStrings(const vector<string> &str, const string &delim) {
 	return ret;
 }
 
+vector<string> JoinStringsMaxWidth(
+	const vector<string> &str, const string &delim, const size_t max_width) {
+	vector<string> ret;
+	auto s = str.begin();
+	if (s == str.end()) {
+		return ret;
+	}
+
+	string line = *s;
+	s++;
+	for (; s != str.end(); s++) {
```

Comment:
S: Why not add this extra functionality as a function to `JoinStrings`, and then give it an empty default? 

Then we don't have to duplicate this?f

Reply:
I will try.

Reply:
True, didn't think of that...